src/components/ProjectField/TextAreaDisplay.tsx

```diff
@@ -21,9 +22,19 @@ const ProjectFieldTextAreaDisplay = ({ label, value }: ProjectFieldProps) => {
         <Typography fontSize={'16px'} lineHeight={'24px'} fontWeight={600} marginBottom={theme.spacing(1)}>
           {label}
         </Typography>
-        <Typography fontSize={'16px'} lineHeight={'24px'} fontWeight={400}>
-          {value}
-        </Typography>
+        {typeof value === 'string' && (
+          <Typography fontSize={'16px'} lineHeight={'24px'} fontWeight={400}>
+            <TextField
+              id=''
+              label=''
```

Comment:
There's no actual `<textarea>` being produced here so it didn't seem like the ID or label would be meaningful. If using empty strings is a massive no-no, I can figure out values to put there.